src/main/java/picard/sam/AbstractAlignmentMerger.java

```diff
@@ -663,7 +662,7 @@ protected void setValuesFromAlignment(final SAMRecord rec, final SAMRecord align
         // If it's on the negative strand, reverse complement the bases
         // and reverse the order of the qualities
         if (rec.getReadNegativeStrandFlag()) {
-            SAMRecordUtil.reverseComplement(rec);
+            rec.reverseComplement();
```

Comment:
Because the default changed, in order for rec.reverseComplement() to remain an in-place operation (instead of making a copy), this should be rec.reverseComplement(true); Even though, I'm planning on changing this in #663, it's probably better to keep the behavior the same as it currently is in this PR.

There's also a usage of SamRecordUtil.reverseComplement in RevertSam. Could you also update that one to rec.reverseComplement(true)?